components/create_post/create_post.jsx

```diff
@@ -557,7 +557,8 @@ export default class CreatePost extends React.Component {
     }
 
     showShortcuts(e) {
-        if ((e.ctrlKey || e.metaKey) && e.keyCode === KeyCodes.FORWARD_SLASH) {
+        const FORWARD_SLASH_KEY = ['/', 191];
```

Comment:
(Just a note)
If we'll be expanding support to keys for all equivalent at `Contants.KeyCodes`, then we might move this to Constants in the future.